server/src/test/java/io/spine/server/event/EventBusShould.java

```diff
@@ -79,7 +86,7 @@ public void setUp() {
 
     private void setUp(@Nullable EventEnricher enricher) {
         final BoundedContext bc = BoundedContext.newBuilder()
-                                                .setMultitenant(true)
+                                                .setMultitenant(false)
```

Comment:
This change was introduced because multitenant environments do not function properly at the moment. 

No tests were affected by this change.